server/lib/PLS/Server/Request/Exit.pm

```diff
@@ -0,0 +1,26 @@
+package PLS::Server::Request::Exit;
+
+use strict;
+use warnings;
+
+use parent 'PLS::Server::Request';
+
+use Scalar::Util qw(blessed);
+
+=head1 NAME
+
+PLS::Server::Request::Exit
+
+=head1 DESCRIPTION
+
+This is a notification message from the client to the server requesting
+that the server exits.
+
+=cut
+
+sub service
+{
+    exit 1;
```

Comment:
Why is this `exit 1`? The Exit request is a graceful exit (or should be) - this should be `exit 0`.

Reply:
Correction - the exit code should be 0 if Shutdown has been sent, 1 otherwise.

>A notification to ask the server to exit its process. The server should exit with success code 0 if the shutdown request has been received before; otherwise with error code 1.

Reply:
Please implement as required